src/Codeception/Module/WebDriver.php

```diff
@@ -1952,7 +1952,9 @@ public function waitForElementNotVisible($element, $timeout = 10)
 
     /**
      * Waits up to $timeout seconds for the given string to appear on the page.
-     * Can also be passed a selector to search in.
+     *
+     * Can also be passed a selector to search in, be as specific as possible when using selectors.
+     * WaitForText() will only watch the first instance of the matching selector / text provided.
```

Comment:
Method names aren't case sensitive, but PSR-2 requires them to be formatted this way.